projects/igniteui-angular/src/lib/services/exporter-common/base-export-service.ts

```diff
@@ -490,7 +490,8 @@ export abstract class IgxBaseExporter {
         const rowArgs = {
             rowData: record.data,
             rowIndex: index,
-            cancel: false
+            cancel: false,
+            owner: record.owner
```

Comment:
Maybe we can use the owner const instead, this way we will have the DEFAULT_OWNER in case the record's one is undefined?